src/features/ScheduleCard.tsx

```diff
@@ -2,17 +2,17 @@ import React, { useState, useContext, useMemo } from 'react';
 import Skeleton from 'react-loading-skeleton';
 import { isSameDay, format, isWithinRange } from 'date-fns';
 import VisuallyHidden from '@reach/visually-hidden';
-import { useCourseSchedule, usePlannerItems } from '../api/student';
-import { useAcademicCalendarEvents } from '../api/events';
-import { UserContext } from '../App';
-import { getNextFiveDays, getDayShortcode, coursesOnDay } from './schedule/schedule-utils';
 import {
   ScheduleCardDayMenu,
   ScheduleCardCourses,
   ScheduleCardAssignments,
   ScheduleCardAcademicCalendar
 } from './schedule';
+import { getNextFiveDays, getDayShortcode, coursesOnDay } from './schedule/schedule-utils';
 import { Header } from './schedule/ScheduleCardStyles';
+import { useAcademicCalendarEvents } from '../api/events';
+import { useCourseSchedule, usePlannerItems } from '../api/student';
+import { UserContext } from '../App';
```

Comment:
Was it necessary to change the order of the imports to make the feature work, or should this file be reverted and left as-is?

Reply:
i changed it so that it was in alphabetical order ... i don't know why i felt the need to do that, but that's the logic behind the mayhem. since i was using another file a lot as a guide, i was spending a lot of time bringing imports over, and having to see if i already imported. .. so i just organized them so it was much faster to see if i had already imported from the file for a different purpose

Reply:
Ah that makes sense.. since the file itself has no functional changes, just revert it and don't include it in the PR